From b8bc671e847e5ffe706b36b05c0e5263f7a3e5d7 Mon Sep 17 00:00:00 2001
From: Byben <heybyben@gmail.com>
Date: Mon, 12 Oct 2026 08:00:00 +0700
Subject: [PATCH 12/14] Add Leica looks icons

Non-Leica builds ship blank placeholder shapes for the Leica looks
selector (white blocks in UI). Replace with the real Leica vector
icons so Vibrant/Authentic/Master/Other entries render properly.
Smali unlock gates are covered by earlier patches; this only supplies
the missing assets.

Assisted-by: LLM
Change-Id: I0ebb831c2aaa2be8ae7b4694aacb4c82ded83655
---
 res/drawable/ic_cv_logo.xml                    |  7 ++++---
 res/drawable/ic_cvtype_item_1_mm.xml           | 18 ++++++++++++++----
 res/drawable/ic_cvtype_item_1_top_mm.xml       | 18 ++++++++++++++----
 res/drawable/ic_cvtype_item_2_mm.xml           | 18 ++++++++++++++----
 res/drawable/ic_cvtype_item_2_top_mm.xml       | 18 ++++++++++++++----
 res/drawable/ic_cvtype_item_master.xml         | 18 ++++++++++++++----
 .../ic_cvtype_item_master_top_menu.xml         | 18 ++++++++++++++----
 res/drawable/ic_cvtype_item_other.xml          | 18 ++++++++++++++----
 res/drawable/ic_cvtype_item_other_top_menu.xml | 18 ++++++++++++++----
 9 files changed, 116 insertions(+), 35 deletions(-)

diff --git a/res/drawable/ic_cv_logo.xml b/res/drawable/ic_cv_logo.xml
index c63db04c..3112692b 100644
--- a/res/drawable/ic_cv_logo.xml
+++ b/res/drawable/ic_cv_logo.xml
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
+<vector android:height="16.0dp" android:width="56.0dp" android:viewportWidth="56.0" android:viewportHeight="16.0"
   xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+    <path android:fillColor="#cd0d0d" android:pathData="M0,0H56V14C56,15.105 55.105,16 54,16H2C0.895,16 0,15.105 0,14V0Z" android:strokeAlpha="0.9" android:fillAlpha="0.72" />
+    <path android:fillColor="#ffffff" android:pathData="M14.165,11.25V4.35H15.175V10.33H18.425V11.25H14.165ZM20.245,11.25V4.35H24.805V5.27H21.255V7.19H24.505V8.11H21.255V10.33H24.805V11.25H20.245ZM26.883,11.25V4.35H27.893V11.25H26.883ZM30.256,9.67V5.93C30.256,4.877 30.78,4.35 31.826,4.35L34.606,4.35L35.116,4.75V5.27L31.836,5.27C31.456,5.27 31.266,5.457 31.266,5.83V9.77C31.266,10.143 31.456,10.33 31.836,10.33H35.116V10.85L34.606,11.25H31.826C30.78,11.25 30.256,10.723 30.256,9.67ZM36.509,11.04L38.719,4.35H40.329L42.529,11.04L42.399,11.25H41.549L41.029,9.61H38.019L37.489,11.25H36.649L36.509,11.04ZM38.309,8.69H40.739L39.629,5.27H39.419L38.309,8.69Z" android:strokeAlpha="0.9" android:fillAlpha="0.9" />
+</vector>
diff --git a/res/drawable/ic_cvtype_item_1_mm.xml b/res/drawable/ic_cvtype_item_1_mm.xml
index c63db04c..df9bde97 100644
--- a/res/drawable/ic_cvtype_item_1_mm.xml
+++ b/res/drawable/ic_cvtype_item_1_mm.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.388,15.76L5.365,15.829L4.039,19.843L4.024,19.889L4.051,19.93L4.135,20.056L4.165,20.1H4.795L4.817,20.03L5.113,19.116H6.773L7.063,20.03L7.085,20.1H7.724L7.753,20.053L7.831,19.927L7.855,19.887L7.842,19.843L6.521,15.829L6.499,15.76H5.388ZM8.311,15.76L8.311,19.052C8.311,19.383 8.393,19.65 8.575,19.833C8.758,20.017 9.023,20.1 9.353,20.1H10.624C10.954,20.1 11.219,20.017 11.401,19.833C11.583,19.65 11.666,19.383 11.666,19.052L11.666,15.76H10.86L10.86,19.112C10.86,19.208 10.836,19.262 10.805,19.293C10.773,19.324 10.717,19.348 10.618,19.348H9.358C9.26,19.348 9.203,19.324 9.172,19.293C9.141,19.262 9.116,19.208 9.116,19.112L9.116,15.76H8.311ZM12.165,15.76V16.512H13.509L13.509,20.1H14.321L14.321,16.512H15.665V15.76H12.165ZM16.194,15.76L16.194,20.1H17.001V18.216H18.745V20.1H19.551V15.76H18.745V17.464L17.001,17.464V15.76H16.194ZM6.535,18.364H5.352L5.943,16.541L6.535,18.364ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_1_top_mm.xml b/res/drawable/ic_cvtype_item_1_top_mm.xml
index c63db04c..df9bde97 100644
--- a/res/drawable/ic_cvtype_item_1_top_mm.xml
+++ b/res/drawable/ic_cvtype_item_1_top_mm.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.388,15.76L5.365,15.829L4.039,19.843L4.024,19.889L4.051,19.93L4.135,20.056L4.165,20.1H4.795L4.817,20.03L5.113,19.116H6.773L7.063,20.03L7.085,20.1H7.724L7.753,20.053L7.831,19.927L7.855,19.887L7.842,19.843L6.521,15.829L6.499,15.76H5.388ZM8.311,15.76L8.311,19.052C8.311,19.383 8.393,19.65 8.575,19.833C8.758,20.017 9.023,20.1 9.353,20.1H10.624C10.954,20.1 11.219,20.017 11.401,19.833C11.583,19.65 11.666,19.383 11.666,19.052L11.666,15.76H10.86L10.86,19.112C10.86,19.208 10.836,19.262 10.805,19.293C10.773,19.324 10.717,19.348 10.618,19.348H9.358C9.26,19.348 9.203,19.324 9.172,19.293C9.141,19.262 9.116,19.208 9.116,19.112L9.116,15.76H8.311ZM12.165,15.76V16.512H13.509L13.509,20.1H14.321L14.321,16.512H15.665V15.76H12.165ZM16.194,15.76L16.194,20.1H17.001V18.216H18.745V20.1H19.551V15.76H18.745V17.464L17.001,17.464V15.76H16.194ZM6.535,18.364H5.352L5.943,16.541L6.535,18.364ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_2_mm.xml b/res/drawable/ic_cvtype_item_2_mm.xml
index c63db04c..39ced14c 100644
--- a/res/drawable/ic_cvtype_item_2_mm.xml
+++ b/res/drawable/ic_cvtype_item_2_mm.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.469,15.76L5.438,15.805L5.354,15.931L5.328,15.971L5.343,16.017L6.614,20.03L6.637,20.1H7.798L7.819,20.03L9.085,16.016L9.1,15.973L9.075,15.934L8.997,15.808L8.968,15.76H8.311L8.288,15.831L7.217,19.313L6.146,15.831L6.124,15.76H5.469ZM9.592,15.76L9.592,20.1H10.398L10.398,15.76H9.592ZM11.315,15.76L11.315,20.1L13.449,20.1C13.779,20.1 14.044,20.017 14.227,19.833C14.409,19.65 14.491,19.383 14.491,19.052L14.491,18.494C14.491,18.289 14.435,18.116 14.313,17.988L14.257,17.937L14.082,17.791L14.229,17.616C14.325,17.501 14.371,17.361 14.371,17.204V16.808C14.371,16.476 14.289,16.21 14.106,16.026C13.924,15.843 13.658,15.76 13.329,15.76H11.315ZM15.254,15.76L15.254,20.1H16.06L16.06,18.516H16.937L17.713,20.045L17.741,20.1H18.379L18.407,20.046L18.479,19.908L18.503,19.862L18.479,19.817L17.783,18.435C17.955,18.378 18.095,18.282 18.194,18.141C18.315,17.968 18.37,17.742 18.37,17.474V16.808C18.37,16.476 18.288,16.21 18.105,16.026C17.923,15.843 17.658,15.76 17.328,15.76H15.254ZM13.443,18.216C13.542,18.216 13.597,18.241 13.629,18.272C13.66,18.304 13.686,18.359 13.686,18.458L13.686,19.112C13.686,19.208 13.661,19.262 13.63,19.293C13.598,19.324 13.542,19.348 13.443,19.348H12.121L12.121,18.216H13.443ZM17.771,18.338L17.641,18.374C17.691,18.364 17.737,18.35 17.781,18.333C17.778,18.334 17.774,18.337 17.771,18.338ZM17.322,16.512C17.421,16.512 17.477,16.536 17.509,16.567C17.54,16.598 17.565,16.652 17.565,16.748V17.522C17.564,17.621 17.539,17.676 17.508,17.708C17.476,17.74 17.421,17.764 17.322,17.764H16.06V16.512H17.322ZM13.323,16.512C13.422,16.512 13.478,16.536 13.51,16.567C13.541,16.598 13.565,16.652 13.565,16.748V17.222C13.565,17.32 13.54,17.377 13.509,17.408C13.477,17.44 13.421,17.464 13.323,17.464H12.121V16.512H13.323ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_2_top_mm.xml b/res/drawable/ic_cvtype_item_2_top_mm.xml
index c63db04c..39ced14c 100644
--- a/res/drawable/ic_cvtype_item_2_top_mm.xml
+++ b/res/drawable/ic_cvtype_item_2_top_mm.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.469,15.76L5.438,15.805L5.354,15.931L5.328,15.971L5.343,16.017L6.614,20.03L6.637,20.1H7.798L7.819,20.03L9.085,16.016L9.1,15.973L9.075,15.934L8.997,15.808L8.968,15.76H8.311L8.288,15.831L7.217,19.313L6.146,15.831L6.124,15.76H5.469ZM9.592,15.76L9.592,20.1H10.398L10.398,15.76H9.592ZM11.315,15.76L11.315,20.1L13.449,20.1C13.779,20.1 14.044,20.017 14.227,19.833C14.409,19.65 14.491,19.383 14.491,19.052L14.491,18.494C14.491,18.289 14.435,18.116 14.313,17.988L14.257,17.937L14.082,17.791L14.229,17.616C14.325,17.501 14.371,17.361 14.371,17.204V16.808C14.371,16.476 14.289,16.21 14.106,16.026C13.924,15.843 13.658,15.76 13.329,15.76H11.315ZM15.254,15.76L15.254,20.1H16.06L16.06,18.516H16.937L17.713,20.045L17.741,20.1H18.379L18.407,20.046L18.479,19.908L18.503,19.862L18.479,19.817L17.783,18.435C17.955,18.378 18.095,18.282 18.194,18.141C18.315,17.968 18.37,17.742 18.37,17.474V16.808C18.37,16.476 18.288,16.21 18.105,16.026C17.923,15.843 17.658,15.76 17.328,15.76H15.254ZM13.443,18.216C13.542,18.216 13.597,18.241 13.629,18.272C13.66,18.304 13.686,18.359 13.686,18.458L13.686,19.112C13.686,19.208 13.661,19.262 13.63,19.293C13.598,19.324 13.542,19.348 13.443,19.348H12.121L12.121,18.216H13.443ZM17.771,18.338L17.641,18.374C17.691,18.364 17.737,18.35 17.781,18.333C17.778,18.334 17.774,18.337 17.771,18.338ZM17.322,16.512C17.421,16.512 17.477,16.536 17.509,16.567C17.54,16.598 17.565,16.652 17.565,16.748V17.522C17.564,17.621 17.539,17.676 17.508,17.708C17.476,17.74 17.421,17.764 17.322,17.764H16.06V16.512H17.322ZM13.323,16.512C13.422,16.512 13.478,16.536 13.51,16.567C13.541,16.598 13.565,16.652 13.565,16.748V17.222C13.565,17.32 13.54,17.377 13.509,17.408C13.477,17.44 13.421,17.464 13.323,17.464H12.121V16.512H13.323ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_master.xml b/res/drawable/ic_cvtype_item_master.xml
index c63db04c..39ced14c 100644
--- a/res/drawable/ic_cvtype_item_master.xml
+++ b/res/drawable/ic_cvtype_item_master.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.469,15.76L5.438,15.805L5.354,15.931L5.328,15.971L5.343,16.017L6.614,20.03L6.637,20.1H7.798L7.819,20.03L9.085,16.016L9.1,15.973L9.075,15.934L8.997,15.808L8.968,15.76H8.311L8.288,15.831L7.217,19.313L6.146,15.831L6.124,15.76H5.469ZM9.592,15.76L9.592,20.1H10.398L10.398,15.76H9.592ZM11.315,15.76L11.315,20.1L13.449,20.1C13.779,20.1 14.044,20.017 14.227,19.833C14.409,19.65 14.491,19.383 14.491,19.052L14.491,18.494C14.491,18.289 14.435,18.116 14.313,17.988L14.257,17.937L14.082,17.791L14.229,17.616C14.325,17.501 14.371,17.361 14.371,17.204V16.808C14.371,16.476 14.289,16.21 14.106,16.026C13.924,15.843 13.658,15.76 13.329,15.76H11.315ZM15.254,15.76L15.254,20.1H16.06L16.06,18.516H16.937L17.713,20.045L17.741,20.1H18.379L18.407,20.046L18.479,19.908L18.503,19.862L18.479,19.817L17.783,18.435C17.955,18.378 18.095,18.282 18.194,18.141C18.315,17.968 18.37,17.742 18.37,17.474V16.808C18.37,16.476 18.288,16.21 18.105,16.026C17.923,15.843 17.658,15.76 17.328,15.76H15.254ZM13.443,18.216C13.542,18.216 13.597,18.241 13.629,18.272C13.66,18.304 13.686,18.359 13.686,18.458L13.686,19.112C13.686,19.208 13.661,19.262 13.63,19.293C13.598,19.324 13.542,19.348 13.443,19.348H12.121L12.121,18.216H13.443ZM17.771,18.338L17.641,18.374C17.691,18.364 17.737,18.35 17.781,18.333C17.778,18.334 17.774,18.337 17.771,18.338ZM17.322,16.512C17.421,16.512 17.477,16.536 17.509,16.567C17.54,16.598 17.565,16.652 17.565,16.748V17.522C17.564,17.621 17.539,17.676 17.508,17.708C17.476,17.74 17.421,17.764 17.322,17.764H16.06V16.512H17.322ZM13.323,16.512C13.422,16.512 13.478,16.536 13.51,16.567C13.541,16.598 13.565,16.652 13.565,16.748V17.222C13.565,17.32 13.54,17.377 13.509,17.408C13.477,17.44 13.421,17.464 13.323,17.464H12.121V16.512H13.323ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_master_top_menu.xml b/res/drawable/ic_cvtype_item_master_top_menu.xml
index c63db04c..39ced14c 100644
--- a/res/drawable/ic_cvtype_item_master_top_menu.xml
+++ b/res/drawable/ic_cvtype_item_master_top_menu.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.469,15.76L5.438,15.805L5.354,15.931L5.328,15.971L5.343,16.017L6.614,20.03L6.637,20.1H7.798L7.819,20.03L9.085,16.016L9.1,15.973L9.075,15.934L8.997,15.808L8.968,15.76H8.311L8.288,15.831L7.217,19.313L6.146,15.831L6.124,15.76H5.469ZM9.592,15.76L9.592,20.1H10.398L10.398,15.76H9.592ZM11.315,15.76L11.315,20.1L13.449,20.1C13.779,20.1 14.044,20.017 14.227,19.833C14.409,19.65 14.491,19.383 14.491,19.052L14.491,18.494C14.491,18.289 14.435,18.116 14.313,17.988L14.257,17.937L14.082,17.791L14.229,17.616C14.325,17.501 14.371,17.361 14.371,17.204V16.808C14.371,16.476 14.289,16.21 14.106,16.026C13.924,15.843 13.658,15.76 13.329,15.76H11.315ZM15.254,15.76L15.254,20.1H16.06L16.06,18.516H16.937L17.713,20.045L17.741,20.1H18.379L18.407,20.046L18.479,19.908L18.503,19.862L18.479,19.817L17.783,18.435C17.955,18.378 18.095,18.282 18.194,18.141C18.315,17.968 18.37,17.742 18.37,17.474V16.808C18.37,16.476 18.288,16.21 18.105,16.026C17.923,15.843 17.658,15.76 17.328,15.76H15.254ZM13.443,18.216C13.542,18.216 13.597,18.241 13.629,18.272C13.66,18.304 13.686,18.359 13.686,18.458L13.686,19.112C13.686,19.208 13.661,19.262 13.63,19.293C13.598,19.324 13.542,19.348 13.443,19.348H12.121L12.121,18.216H13.443ZM17.771,18.338L17.641,18.374C17.691,18.364 17.737,18.35 17.781,18.333C17.778,18.334 17.774,18.337 17.771,18.338ZM17.322,16.512C17.421,16.512 17.477,16.536 17.509,16.567C17.54,16.598 17.565,16.652 17.565,16.748V17.522C17.564,17.621 17.539,17.676 17.508,17.708C17.476,17.74 17.421,17.764 17.322,17.764H16.06V16.512H17.322ZM13.323,16.512C13.422,16.512 13.478,16.536 13.51,16.567C13.541,16.598 13.565,16.652 13.565,16.748V17.222C13.565,17.32 13.54,17.377 13.509,17.408C13.477,17.44 13.421,17.464 13.323,17.464H12.121V16.512H13.323ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_other.xml b/res/drawable/ic_cvtype_item_other.xml
index c63db04c..df9bde97 100644
--- a/res/drawable/ic_cvtype_item_other.xml
+++ b/res/drawable/ic_cvtype_item_other.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.388,15.76L5.365,15.829L4.039,19.843L4.024,19.889L4.051,19.93L4.135,20.056L4.165,20.1H4.795L4.817,20.03L5.113,19.116H6.773L7.063,20.03L7.085,20.1H7.724L7.753,20.053L7.831,19.927L7.855,19.887L7.842,19.843L6.521,15.829L6.499,15.76H5.388ZM8.311,15.76L8.311,19.052C8.311,19.383 8.393,19.65 8.575,19.833C8.758,20.017 9.023,20.1 9.353,20.1H10.624C10.954,20.1 11.219,20.017 11.401,19.833C11.583,19.65 11.666,19.383 11.666,19.052L11.666,15.76H10.86L10.86,19.112C10.86,19.208 10.836,19.262 10.805,19.293C10.773,19.324 10.717,19.348 10.618,19.348H9.358C9.26,19.348 9.203,19.324 9.172,19.293C9.141,19.262 9.116,19.208 9.116,19.112L9.116,15.76H8.311ZM12.165,15.76V16.512H13.509L13.509,20.1H14.321L14.321,16.512H15.665V15.76H12.165ZM16.194,15.76L16.194,20.1H17.001V18.216H18.745V20.1H19.551V15.76H18.745V17.464L17.001,17.464V15.76H16.194ZM6.535,18.364H5.352L5.943,16.541L6.535,18.364ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
diff --git a/res/drawable/ic_cvtype_item_other_top_menu.xml b/res/drawable/ic_cvtype_item_other_top_menu.xml
index c63db04c..df9bde97 100644
--- a/res/drawable/ic_cvtype_item_other_top_menu.xml
+++ b/res/drawable/ic_cvtype_item_other_top_menu.xml
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-  xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/white" />
-</shape>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="40.0dp"
+    android:width="40.0dp"
+    android:viewportWidth="40.0"
+    android:viewportHeight="40.0">
+    <group
+        android:scaleX="0.9"
+        android:scaleY="0.9"
+        android:translateX="9.2"
+        android:translateY="9.2">
+        <path android:fillColor="#ffffff" android:pathData="M23,22L1,22L1,2L23,2L23,22ZM5.388,15.76L5.365,15.829L4.039,19.843L4.024,19.889L4.051,19.93L4.135,20.056L4.165,20.1H4.795L4.817,20.03L5.113,19.116H6.773L7.063,20.03L7.085,20.1H7.724L7.753,20.053L7.831,19.927L7.855,19.887L7.842,19.843L6.521,15.829L6.499,15.76H5.388ZM8.311,15.76L8.311,19.052C8.311,19.383 8.393,19.65 8.575,19.833C8.758,20.017 9.023,20.1 9.353,20.1H10.624C10.954,20.1 11.219,20.017 11.401,19.833C11.583,19.65 11.666,19.383 11.666,19.052L11.666,15.76H10.86L10.86,19.112C10.86,19.208 10.836,19.262 10.805,19.293C10.773,19.324 10.717,19.348 10.618,19.348H9.358C9.26,19.348 9.203,19.324 9.172,19.293C9.141,19.262 9.116,19.208 9.116,19.112L9.116,15.76H8.311ZM12.165,15.76V16.512H13.509L13.509,20.1H14.321L14.321,16.512H15.665V15.76H12.165ZM16.194,15.76L16.194,20.1H17.001V18.216H18.745V20.1H19.551V15.76H18.745V17.464L17.001,17.464V15.76H16.194ZM6.535,18.364H5.352L5.943,16.541L6.535,18.364ZM2.6,14L21.4,14L21.4,3.6L2.6,3.6L2.6,14Z" android:fillAlpha="0.8" />
+        <path android:fillColor="#ffffff" android:pathData="M4.539,11V6.86H5.145V10.448H7.095V11H4.539ZM7.767,11V6.86H10.503V7.412H8.373V8.564H10.323V9.116H8.373V10.448H10.503V11H7.767ZM11.33,11V6.86H11.936V11H11.33ZM12.934,10.052V7.808C12.934,7.176 13.248,6.86 13.876,6.86H15.544L15.85,7.1V7.412H13.882C13.654,7.412 13.54,7.524 13.54,7.748V10.112C13.54,10.336 13.654,10.448 13.882,10.448H15.85V10.76L15.544,11H13.876C13.248,11 12.934,10.684 12.934,10.052ZM16.265,10.874L17.591,6.86H18.557L19.877,10.874L19.799,11H19.289L18.977,10.016H17.171L16.853,11H16.349L16.265,10.874ZM17.345,9.464H18.803L18.137,7.412H18.011L17.345,9.464Z" android:fillAlpha="0.8" />
+    </group>
+</vector>
-- 
2.54.0

